### PR TITLE
Several grafana fixes

### PR DESCRIPTION
**Pull request by fgimenez**
c&p error, i noticed that the service was not accessible from https://grafana.ci.kubevirt.io/

/cc @dhiller @rmohr 

Signed-off-by: Federico Gimenez <fgimenez@redhat.com>

### CODE REVIEW

**Comment by fgimenez** [comment]
Several things are addressed in this PR:

* fixed deployment code to remove new lines from secrets
* fixed prometheus service URL in grafana data source definition
* fixed org_name entry in grafana.ini for anonymous auth 
* c&p error in the ingress service selector, i noticed that the service was not accessible from https://grafana.ci.kubevirt.io/

File: github/ci/services/prometheus-stack/manifests/grafana.yaml [hunk 1]
```diff
@@ -141,7 +141,7 @@ data:
     provisioning = /etc/grafana/provisioning
     [auth.anonymous]
     enabled = true
-    org_name = KubeVirt community
+    org_name = Main Org.
```
Comment (dhiller) [inline github/ci/services/prometheus-stack/manifests/grafana.yaml:144]: This looks odd?

**Review by dhiller — APPROVED** [review]
/lgtm
/approve
/hold

Besides the org_name looking odd?

File: github/ci/services/prometheus-stack/manifests/grafana.yaml (hunk 1 above)
Comment (fgimenez) [reply github/ci/services/prometheus-stack/manifests/grafana.yaml:144]: Yep, but it is the value of the default org created, without it there's an "unknown org" error in the logs and people not already logged get to the logging screen (not direct access given that they can be assigned to a default org). This is the same value that we use in the existing deployment https://github.com/kubevirt/project-infra/blob/master/github/ci/prometheus/templates/grafana-config.yaml#L16

**Pull request merged**